core/array/pack/l_spec.rb

```diff
@@ -29,7 +29,7 @@
     it_behaves_like :array_pack_32bit_be, 'L>'
   end
 
-  guard -> { platform_is wordsize: 32 or platform_is :mingw32 } do
+  guard -> { platform_is wordsize: 32 } do
```

Comment:
Could you simplify to just `platform_is wordsize: 32 do`?

Also, this used to fail on AppVeyor's Ruby IIRC (that's the reason of these guards), is the wordsize seen as 32 bits on MinGW on 64-bit Windows now?

Reply:
@eregon

Sorry about leaving the guard.  There's no Appveyor testing on MinGW 32 bit.

>  is the wordsize seen as 32 bits on MinGW on 64-bit Windows now?

Off hand, I don't know.  The MSYS2/RubyInstaller2 builds (>= Ruby 2.4) are built with much newer toolchains.  Appveyor is only testing >= Ruby 2.4 64 bit, but Travis is testing 2.3.  

I'll check them with 32 bit versions of Ruby 2.3 & 2.4, and let you know the results.  Should these remain for platforms other than Windows?

Reply:
> Should these remain for platforms other than Windows?

Yes, the wordsize guards are needed, but indeed maybe we can drop the exception of mingw32, please verify. I think I originally had the guards to fix failures on AppVeyor with 32-bit Ruby.

Reply:
Forgive me, I'm confused.

> I think I originally had the guards to fix failures on AppVeyor with 32-bit Ruby.

Umm, 'or', not 'and'.  Hence, I assumed they needed the guard on non-Windows 32 bit builds.

I just ran core files as they are after this PR, and core/array/pack/l_spec.rb reported the same number of examples and expectations when tested on 2.4.5 32 bit and trunk 64 bit.

Ruby 2.3 32 bit is something I think we should forget.  23 failures on core.  I haven't tried 2.3 64 bit.

I'll remove the guard statements...